src/main/java/io/codelex/flightplanner/flightplan/AdminCommandsController.java

```diff
@@ -0,0 +1,48 @@
+package io.codelex.flightplanner.flightplan;
+
+import io.codelex.flightplanner.flightplan.domain.Flight;
+import io.codelex.flightplanner.flightplan.dto.FlightRequest;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+public class AdminCommandsController {
+    private FlightPlanService flightPlanService;
+
+    public AdminCommandsController(FlightPlanService flightPlanService) {
+
+        this.flightPlanService = flightPlanService;
+    }
+
+
+    public FlightPlanService getFlightPlanService() {
+        return flightPlanService;
+    }
+
+    public void setFlightPlanService(FlightPlanService flightPlanService) {
+        this.flightPlanService = flightPlanService;
+    }
+
+    @PutMapping(value = "/admin-api/flights")
+    @ResponseStatus(HttpStatus.CREATED)
+    public Flight addFlight(@Valid @RequestBody FlightRequest flightRequest) {
+        Flight flight = this.flightPlanService.setID(flightRequest);
```

Comment:
ID uzstādīšanu labāk liec iekš paša service, nav īsti jēgas šeit uzstādīt ID un tad padot ar visu ID servisam. Tad drīzāk padod bez ID un uzstādi to iekš service.